### ISSUE TITLE
Add documentation readthedocs.io like python-consul2

### ISSUE THREAD

**Issue by rhuddleston** [criteo/py-consul#78]
Would it be possible to publish documentation similar to what python-consul2 had? https://python-consul2.readthedocs.io/en/latest/    

python-consul has similar https://python-consul.readthedocs.io/en/latest/

**Comment by rhuddleston** [comment]
another thing maybe to consider is a small section on code migration from python-consul or python-consul2 to py-consul.

**Comment by zffocussss** [comment]
I agree with @rhuddleston ,as a well-written documentation website is very helpful for users.

**Comment by mbrulatout** [comment]
Hi there,
Sorry for the delay, we'll look into this !

**Comment by mbrulatout** [comment]
https://criteo.github.io/py-consul/ should do the trick
